tests/robot-tests/tests/admin_and_public_2/bau/publish_release_and_amend.robot

```diff
@@ -339,6 +353,15 @@ Verify Test text accordion section contains correct text
     user waits until parent contains element    ${section}    xpath:.//p[text()="Some test text!"]
     user closes accordion section    Test text    id:content
 
+Verify embedded dashboard accordion section contains dashboard
+    user opens accordion section    Test embedded dashboard section    id:content
+    ${section}=    user gets accordion section content element    Test embedded dashboard section    id:content
+    user waits until parent contains element    ${section}    xpath:.//iframe[@title="Test embedded dashboard title"]
+
+    select frame    xpath://iframe[@title="Test embedded dashboard title"]
+    user waits until h1 is visible    DfE Analytical Services R-Shiny data dashboard template (h1)    90
```

Comment:
Just wondering why we've got such a long wait here?

Reply:
I think I had an instance when it failed once - I believe because it was taking a long time to load. I'll change it back to wait small.